lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -207,28 +199,19 @@ public APIGatewayProxyResponseEvent handleRequest(
                 LOGGER.info(message);
             }
 
-            return ApiGatewayResponseGenerator.proxyJsonResponse(HttpStatus.SC_OK, JOURNEY_NEXT);
-        } catch (HttpResponseExceptionWithErrorBody e) {
-            return ApiGatewayResponseGenerator.proxyJsonResponse(
-                    e.getResponseCode(), e.getErrorBody());
+            return JOURNEY_NEXT;
         } catch (ParseException e) {
             LOGGER.error("Unable to parse existing credentials", e);
-            return ApiGatewayResponseGenerator.proxyJsonResponse(
-                    HttpStatus.SC_INTERNAL_SERVER_ERROR,
-                    ErrorResponse.FAILED_TO_PARSE_ISSUED_CREDENTIALS);
+            return JOURNEY_ERROR;
         } catch (CiRetrievalException e) {
             LOGGER.error("Error when fetching CIs from storage system", e);
-            return ApiGatewayResponseGenerator.proxyJsonResponse(
-                    HttpStatus.SC_INTERNAL_SERVER_ERROR, ErrorResponse.FAILED_TO_GET_STORED_CIS);
+            return JOURNEY_ERROR;
         } catch (UnknownEvidenceTypeException e) {
             LOGGER.error("Unable to determine type of credential", e);
-            return ApiGatewayResponseGenerator.proxyJsonResponse(
-                    HttpStatus.SC_INTERNAL_SERVER_ERROR,
-                    ErrorResponse.FAILED_TO_DETERMINE_CREDENTIAL_TYPE);
+            return JOURNEY_ERROR;
         } catch (SqsException e) {
             LOGGER.error("Failed to send audit event to SQS queue", e);
-            return ApiGatewayResponseGenerator.proxyJsonResponse(
-                    HttpStatus.SC_INTERNAL_SERVER_ERROR, ErrorResponse.FAILED_TO_SEND_AUDIT_EVENT);
+            return JOURNEY_ERROR;
```

Comment:
Do we want to return journey error in these cases, or do something similar to https://github.com/alphagov/di-ipv-core-back/blob/843d74342b66d3e48536f07a263abbf685f5ce76/lambdas/validate-oauth-callback/src/main/java/uk/gov/di/ipv/core/validateoauthcallback/ValidateOAuthCallbackHandler.java#L198 ?

Reply:
We need to return an object containing `{"journey":"/journey/error"}` however it would be nice to capture the error information as well for debugging issues as this is captured in the express step function. Currently the `generateErrorOutputMap` doesn't return the `journey` field. However, I think I'll create an additional generateErrorOutputMap which returns both the journey as well as the debug status information.

Reply:
Functionality now added